### ISSUE TITLE
Recommendations from ASSESSING BIAS IN STANDARDS AND  CURRICULAR MATERIALS 

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#629]
Suggested to dos from @schanzer & @flannery-denny from [this framework](https://drive.google.com/drive/u/0/folders/15VDSBSkWWivBIG6MlvZn1L89-PQIusrj)

1.1 Support students to understand, investigate, and determine how the implicit cultural assumptions, frames of references, perspectives, and biases within a discipline influence the ways in which knowledge is constructed within it (Banks, 1997).

- be more explicit about how we address this in DS 
- [x] #650

1.2. Support students utilizing collaborative pro- cesses with diverse people in knowledge con- struction to produce new understanding or knowledge that would exceed something that anyone alone could not achieve (Oksanen, 2017).

- We use teamwork and notice/wonder throughout our materials. This touches upon the notion of "new understanding that would exceed something that anyone alone could not achieve", but we can do a LOT more here. 
- Think about when and why we're suggesting collaboration. We offer strategies for collaborations. We have no control over diversity of groups. Not sure how often the collaborative experience exceeds what someone could do on their own. 
- [x] #729

2.3 Are inclusive and reflect the cultural repertoire, practices and contributions of diverse communities in ways both traditional and evolving (Ladson-Billings, 1995; Moll, Amanti, Neff, & Gonzalez, 1992; Sleeter, 2005; Duncan- Andrade, 2007; Gay, 2010; Paris, 2012; Paris & Alim, 2014).

- [x] #649

3.1 Promote or provoke students asking critical questions about societal status quo (Freire, 1970; Duncan-Andrade, 2007).

- [x] #739

3.3 Support students in identifying and using tools and knowledge resources from multiple communities for critiquing the status quo and empowering students to make decisions that will lead to social change towards a just community (Stovall, 2006; Gay, 2010; Aronson & Laughter 2015).

- We are giving them opportunities to engage with datasets that allow for these conversations but offer no scaffolding for taking action toward social change
- [x] #737 

3.4 Liberate students to express one’s emotions, desires and opinions constructively (Duncan- Andrade, 2007; Ladson-Billings, 1994).

- [x] #1105

**Curricular Materials**
2.2 Ableist ideologies that depict people with dis/ abilities as disadvantaged or that they only serve as inspiration are challenged. People with dis/ abilities are depicted in terms of their careers, contributions to society, and active members of communities.

- [x] Add depictions of differently-abled people in our materials. _Calling this done by way of computing needs all voices lesson._

3.2 Curricular materials present history from the perspective of multiple groups of people, inclusive in the general formatting of the text. The text does not “feature” stories of marginalized groups, but rather centers their stories as part of a general, encompassing narrative.

- [x] This would be a nice feature to add to DS, as part of the lesson on bias - See Who are Programmers Lesson and Pioneer Posters

**Comment by flannery-denny** [comment]
[girls who code report with footnotes for data sources](https://www.accenture.com/_acnmedia/PDF-150/Accenture-Cracking-The-Gender-Code-Report.pdf#zoom=50)

Footnotes
1.[http://www.bls.gov/cps/tables.htm]( http://www.bls.gov/cps/tables.htm) Source: BLS - Labor Force Statistics from the Current Population Survey
2. [http://www.aauw.org/2015/03/11/is-computing-just-for-men](http://www.aauw.org/2015/03/11/is-computing-just-for-men)
3. [http://www.npr.org/sections/money/2014/10/21/357629765/when-women-stopped-coding](http://www.npr.org/sections/money/2014/10/21/357629765/when-women-stopped-coding)
4. [http://www.bls.gov/cps/tables.htm Source: Accenture Research calculations based on BLS](http://www.bls.gov/cps/tables.htm Source: Accenture Research calculations based on BLS) - Labor Force Statistics from the Current Population Survey; Table 11 - 2011 to 2015 saw 4.6% growth in computing jobs v 1.5% growth in other jobs
5. [http://robomatter.com/wp-content/uploads/2015/12/Computer-Science-Education-Week-Infographic-2-blue.jpg](http://robomatter.com/wp-content/uploads/2015/12/Computer-Science-Education-Week-Infographic-2-blue.jpg)
6. National Student Clearinghouse Research Center [https://nscresearchcenter.org/snapshotreport-degreeattainment15](https://nscresearchcenter.org/snapshotreport-degreeattainment15)
7. [https://www.whitehouse.gov/sites/default/files/microsites/ostp/imageofstemdepictiondoc_02102016_clean.pdf](https://www.whitehouse.gov/sites/default/files/microsites/ostp/imageofstemdepictiondoc_02102016_clean.pdf)
8.[ http://www3.weforum.org/docs/WEF_Future_of_Jobs.pdf]( http://www3.weforum.org/docs/WEF_Future_of_Jobs.pdf)
9. http://www3.weforum.org/docs/WEF_Future_of_Jobs.pdf
10. The IT–BPM Sector in India, Strategic Review 2015, Nasscom:[http://www.nasscom.in/download/summary_file/fid/110170NASSCOM](http://www.nasscom.in/download/summary_file/fid/110170NASSCOM)
11. [https://www.accenture.com/il-en/~/media/Accenture/Conversion-Assets/DotCom/Documents/Global/PDF/Technology_6/
Accenture-The-Looming-Global-Analytics-Talent-Mismatch-in-Banking.pdf](https://www.accenture.com/il-en/~/media/Accenture/Conversion-Assets/DotCom/Documents/Global/PDF/Technology_6/
Accenture-The-Looming-Global-Analytics-Talent-Mismatch-in-Banking.pdf)
12. [https://www.accenture.com/us-en/gender-equality-research-2016](https://www.accenture.com/us-en/gender-equality-research-2016)
13.[ https://www.whitehouse.gov/sites/default/files/microsites/ostp/imageofstemdepictiondoc_02102016_clean.pdf]( https://www.whitehouse.gov/sites/default/files/microsites/ostp/imageofstemdepictiondoc_02102016_clean.pdf)
14. [http://www.usatoday.com/story/tech/news/2016/03/01/florida-senate-approves-making-coding-foreign-language/81150796](http://www.usatoday.com/story/tech/news/2016/03/01/florida-senate-approves-making-coding-foreign-language/81150796)
15. [http://www.edweek.org/ew/articles/2016/06/09/arkansas-school-coding-initiative-centers-on-teacher-pd.html](http://www.edweek.org/ew/articles/2016/06/09/arkansas-school-coding-initiative-centers-on-teacher-pd.html) and http://time.
com/4203277/computer-science-education/
16. [http://www.sfgate.com/education/article/Tech-shift-More-women-in-computer-science-classes-5243026.php](http://www.sfgate.com/education/article/Tech-shift-More-women-in-computer-science-classes-5243026.php)
17. [http://readwrite.com/2014/09/02/women-in-computer-science-why-so-few/](http://readwrite.com/2014/09/02/women-in-computer-science-why-so-few/)
18. [https://higheredtoday.org/2016/03/02/harvey-mudd-college-confronts-lack-of-female-computer-science-majors/](https://higheredtoday.org/2016/03/02/harvey-mudd-college-confronts-lack-of-female-computer-science-majors/)
19. [https://newsroom.accenture.com/news/female-board-members-far-more-likely-than-male-board-members-to-have-professionaltechnology-experience-accenture-research-finds.htm](https://newsroom.accenture.com/news/female-board-members-far-more-likely-than-male-board-members-to-have-professionaltechnology-experience-accenture-research-finds.htm)
20. Adapted from [https://blogs.microsoft.com/jobs/international/microsoftleadershareswisdomeongrowingyourcareer/
#hJjdapBT54zoCTpm.99](https://blogs.microsoft.com/jobs/international/microsoftleadershareswisdomeongrowingyourcareer/
#hJjdapBT54zoCTpm.99)
21. [http://www.roguelynn.com/about/](http://www.roguelynn.com/about/)

**Comment by flannery-denny** [comment]
@schanzer I am proposing a new core lesson. **Who are programmers? What do they do? And why do we need more?**
Lmk if I have your approval to start writing it.

Objectives - Students will understand:

- There are programmers who look like me and I can relate to.
- Programming is creative and collaborative.
- Coding is useful in in a wide range of fields.
- There are more well-paid jobs that require programming experience than people trained for those jobs.
- People who wrote the programs that we use every day started out learning to write basic code.
- I can be a programmer if I put my mind to it.

Initial Component Ideas:

- [code.org video](https://youtu.be/nKIu9yen5nc ) - notice & wonder
- Pioneer posters & data on expanding employment gap - gallery walk - find one to present to the class.
- windows & mirrors activity (building on Emily Styles framing from SEED) - journal prompt and/or conversation with a partner: identify an image or a moment in the film that is a mirror for you, reflecting some part of your identity or experience of the world. Describe in what ways it is a mirror. Then, identify and write about an image or moment in the film that is a window for you, that lets you look out and understand someone else’s experiences more deeply.

I'd like to find another resources that gets at collaboration more. 

@retabak says "as a teacher, I would be VERY excited about this lesson."

**Comment by flannery-denny** [comment]
@schanzer Pushed a lesson draft. Feel free to edit. Let me know if you think its ready for a slide deck. 

Also I'm getting warnings about it not having any practices. Unless we want to add social justice practices to our standards list, I'm not seeing any to connect this lesson to. Is there a way to tell Dorai's warning system to ignore certain lessons?

**Comment by schanzer** [comment]
@flannery-denny Made some small changes. Let me know if you're ok with them? I think it's ready for the deck. 

Would you put this lesson in all the pathways? If so, where?

I think it's ok to be reminded that this lesson has no practices. Is this severely annoying?

**Comment by flannery-denny** [comment]
@schanzer 
The problem with persistent warnings that we consistently ignore is that it leads to us ignoring warning we should pay attention to.

**Comment by schanzer** [comment]
For me it's a matter of degree: I have enough brain space to remember that one or two warnings are meant to be ignored, but beyond that I just start ignoring everything. But if this morning is bothering you, that's reason enough to take action.

I do like the idea of adding social justice practices to our framework, but I don't have a clear sense for whether or not that's going to be a massive amount of work. But perhaps some overestimating. Is there a list of best practices that you trust, in this regard? Would it be easy to add them?￼￼

**Comment by retabak** [comment]
@flannery-denny @schanzer 

Love this lesson. I have no changes to offer for the existing content.

That said: I can't help but think about what I perceive to be missing from the lesson , which is acknowledgement that the tech industry has a problem with diversity. I *really* like the positive spin (programmers look like you! programming is creative!) -- but I think I (as a teacher) would want to talk to my kids about how we've got a long way to go, and that white males are way over-represented.  

Depending on the poster content, I suppose some of the windows / mirrors reflections could begin to get at that? Not sure.

**Comment by flannery-denny** [comment]
@schanzer These are the standards that the new MSSJ math book had us work with & reference.
[sj standards](https://www.learningforjustice.org/sites/default/files/2017-06/TT_Social_Justice_Standards_0.pdf
![Screen Shot 2021-11-22 at 2 38 11 PM](https://user-images.githubusercontent.com/66691020/142931901-b98838e0-186a-4a30-b297-5ebb41366114.png)
)

@retabak I appreciate what you're saying about diversity in tech and can't focus any attention on this in the coming week. I'm not sure it's within the scope of the lesson, but I have been and will continue to think about what materials are out there and how to share them with students in a way that ensures teachers without their own framing for the work are supported in empowering students rather than demoralizing them.

(ps. for comic relief I just typed "demolarizing" by mistake

**Comment by retabak** [comment]
@flannery-denny - agreed, this makes so much sense. The risk of demoralizing students (or demolarizing them, lol) is real and definitely something we want to avoid. I'll think on it too and let you know if I have any brilliant ideas!

**Comment by schanzer** [comment]
@flannery-denny these practices look _really_ useful. I've added a dictionary file for all of them, and linked that dictionary into the master dictionary Dorai has. Now, the `lessons-and-practices.rkt` dictionary can be modified to add alignment to the social justice practices as well.

**Comment by retabak** [comment]
I was going to do no BS work today, but then ---

I encountered [this article](https://betterprogramming.pub/why-diversity-matters-more-in-programming-ad6c5d3d2641) and am wondering if it might have a place in the lesson you just wrote?  Like, maybe the teacher splits kids into groups and each group brainstorms ways / examples of how each statement is true, based on what they learned in the video and from the posters?

Or.... we could frame a discussion around "[the ketchup question](https://thriveglobal.com/stories/innovative-inclusive-workplace-employee-happiness-ketchup-question-analogy/)" (which I first learned about on a great [Reply All episode](https://gimletmedia.com/shows/reply-all/76h54l) a few years ago).

These approaches both feel way more uplifting / positive than just sharing with kids that THERE IS A VERY REAL PROBLEM, WHAT ARE WE GONNA DO!?! The "ketchup question" could even just be added in as an optional extension question / discussion.
